src/cljs/ataru/virkailija/application/application_subs.cljs

```diff
@@ -266,35 +266,47 @@
   (or (not only-identified?)
       (not (-> application :person :yksiloity))))
 
+(defn- state-filter
+  [states states-to-include default-state-name hakukohteet]
+  (or
+   (not (empty? (clojure.set/intersection
+                 states-to-include
+                 (set states))))
+   (and
+    (contains? states-to-include default-state-name)
+    (or
```

Comment:
Pelkkä `(< (count states) (count hakukohteet)` pitäis riittää.

Reply:
Kokeillaan

Reply:
:+1: